scripts/composer/post-install.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/sh
 
+DOCUMENTROOT=web
```

Comment:
The problem with supporting a config file on the script side is that there is no good way to get composer/installers to read the config value when setting the location of drupal/core in composer.json.